app/controllers/users_controller.rb

```diff
@@ -1,3 +1,36 @@
 class UsersController < ApplicationController
-  def new; end
+  # index show new edit create update destroy
+  def index; end
+
+  def show
+    @user = User.find_by id: params[:id]
+    raise ActionController::RoutingError, t(".notFound") if @user.blank?
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def edit; end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t "welcome"
+      redirect_to controller: "users", action: "show", id: @user.id
```

Comment:
```suggestion
      redirect_to @user
```
Sử dụng như này được không em

Reply:
Ban đầu em để kiểu viết tắt kia nhưng bị lỗi không redirect được anh. Nên em mới phải chuyển sang viết đầy đủ.

Reply:
check lại routes để tìm path tương ứng nhé

dùng rake routes

Reply:
chưa fix